examples/edge-functions/supabase/functions/kysely-postgres/index.ts

```diff
@@ -0,0 +1,90 @@
+// Follow this setup guide to integrate the Deno language server with your editor:
+// https://deno.land/manual/getting_started/setup_your_environment
+// This enables autocomplete, go to definition, etc.
+
+import { serve } from 'std/server'
+import { Pool } from 'postgres'
+import {
+  Kysely,
+  Generated,
+  PostgresAdapter,
+  PostgresIntrospector,
+  PostgresQueryCompiler,
+} from 'kysely'
+import { PostgresDriver } from './DenoPostgresDriver.ts'
+
+console.log(`Function "kysely-postgres" up and running!`)
+
+interface AnimalTable {
+  id: Generated<bigint>
+  animal: string
+  created_at: Date
+}
+
+// Keys of this interface are table names.
+interface Database {
+  animals: AnimalTable
+}
+
+// Create a database pool with one connection.
+const pool = new Pool(
+  {
+    tls: { caCertificates: [Deno.env.get('DB_SSL_CERT')!] },
+    database: 'postgres',
+    hostname: 'db.bljghubhkofddfrezkhn.supabase.co',
```

Comment:
should the hostname come from an env variable?

Reply:
Yes, good catch, thanks.